### PR TITLE
NNS1-2851: Remove SnsTransactionModal

### PR DESCRIPTION
**Pull request by lmuntaner**
# Motivation

Unify wallet implementation.

In this PR, remove SnsTransactoinModal and use IcrcTokenTransactionModal instead.

# Changes

* Add parameters `universeId` to `IcrcTokenTransactionModal` to be used for the TransactionModal and `ledgerCanisterId` is used for the action.
* Pass the `universeId` parameter in the usages of `IcrcTokenTransactionModal`.
* Use `IcrcTokenTransactionModal` in the tokens +page.svelte.
* Remove SnsTransactionModal.

# Tests

* Remove SnsTransactionModal spec file and page object.
* Fix api expect in tokens page spec file.
* Use the icrc transaction modal in the `TokensRoutePo` for sns transactions.

# Todos

- [ ] Add entry to changelog (if necessary).

Not necessary.

### CODE REVIEW

**Comment by lmuntaner** [comment]
Current dependencies on/for this PR:
* `main`
  * **PR #4492** <a href="https://app.graphite.dev/github/pr/dfinity/nns-dapp/4492?utm_source=stack-comment-icon" target="_blank">Graphite</a>  👈

This [stack of pull requests](https://stacking.dev/?utm_source=stack-comment) is managed by [Graphite](https://app.graphite.dev/github/pr/dfinity/nns-dapp/4492?utm_source=stack-comment).

**Comment by lmuntaner** [comment]
@dskloetd please review

File: frontend/src/lib/modals/accounts/SnsTransactionModal.svelte [hunk 1]
```diff
@@ -1,92 +0,0 @@
-<script lang="ts">
-  import { createEventDispatcher } from "svelte";
-  import { startBusy, stopBusy } from "$lib/stores/busy.store";
-  import { i18n } from "$lib/stores/i18n";
-  import { toastsSuccess } from "$lib/stores/toasts.store";
-  import type { NewTransaction } from "$lib/types/transaction";
-  import TransactionModal from "$lib/modals/transaction/TransactionModal.svelte";
-  import { replacePlaceholders } from "$lib/utils/i18n.utils";
-  import { snsTransferTokens } from "$lib/services/sns-accounts.services";
```
Comment (dskloetd) [inline frontend/src/lib/modals/accounts/SnsTransactionModal.svelte:9]: Is `snsTransferTokens` now unused? Will there be a follow-up PR where we delete a lot code?

**Review by dskloetd — APPROVED** [review]
This is great!

File: frontend/src/lib/modals/accounts/SnsTransactionModal.svelte (hunk 1 above)
Comment (lmuntaner) [reply frontend/src/lib/modals/accounts/SnsTransactionModal.svelte:9]: Yes, more PRs might come now.

I need to take a look, but you unblocked quite a few refactors by moving to icrcAccounts.

**Pull request merged**